modules/persistent-storage-csi-smb-cifs-create-sc.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-smb-cifs.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="persistent-storage-csi-smb-cifs-create-sc_{context}"]
+= Creating a storage class for CIFS/SMB
+
+After installing the Operator, you should create a storage class for dynamic provisioning of Common Internet File System (CIFS) dialect/Server Message Block (SMB) protocol volumes.
+
+.Prerequisites
+* You are logged in to the running {product-title} cluster.
+
+* You have SMB server installed and know its:
```

Comment:
Per the [ISG](https://www.ibm.com/docs/en/ibm-style?topic=format-lists#lead-in-wording) we should use complete sentences to introduce lists: 
```suggestion
* You have SMB server installed and know the following information about the server:
```

Feel free to change up some of the wording if you think of a better way to phrase it.